lib/common.bash

```diff
@@ -61,7 +61,9 @@ get_docker_kata_path(){
 extract_kata_env(){
 	local toml
 	local rpath=$(get_docker_kata_path "$RUNTIME")
-	rpath=$(command -v "$rpath")
+	if [ -n "$rpath" ]; then
+		rpath=$(command -v "$rpath")
+	fi
 
 	# If we can execute the path handed back to us
 	if [ -x "$rpath" ]; then
```

Comment:
ah, I thought I'd already done something similar ;-) - but, with the new addition, I guess not good/early enough now...